Source/Extensions/Blazorise.DataGrid/Contexts/DetailRowTriggerContext.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Blazorise.DataGrid
+{
+    /// <summary>
+    /// Holds the DetailRowTrigger context configuration.
+    /// </summary>
+    /// <typeparam name="TItem"></typeparam>
+    public class DetailRowTriggerContext<TItem>
```

Comment:
Maybe this would be better named `DetailRowTriggerEventArgs`? The Context suffix is mostly on parameters that are event callbacks.

Reply:
Well Action/Func as Parameters, pretty much do work like callbacks in blazor if you think about it, just that they do not have the `StateHasChanged` implicit call.
Eitherway I don't really mind either I guess. I'll make the change.

Reply:
We might actually consider a standard way of naming stuff. For example, `Context` can be reserved for render fragments(templates), and EventArgs can be for everything else, `Action`, `Func`, or `EventCallback`.

Reply:
Might be worth having a technical doc with this kind of stuff... our conventions / blazorise best practices etc... Which would also be in the how to contribute.